animations/lib/src/misc/curved_animation.dart

```diff
@@ -0,0 +1,156 @@
+      reverseCurve: selectedReverseCurve.curve,
+    );
+    animationRotation = Tween<double>(
+      begin: 0,
+      end: 2 * math.pi,
+    ).animate(curvedAnimation)
+      ..addListener(() {
+        setState(() {});
+      })
+      ..addStatusListener((status) {
+        if (status == AnimationStatus.completed) {
+          controller.reverse();
+        }
+      });
+    animationTranslation = Tween<Offset>(
+      begin: Offset(-1, 0),
+      end: Offset(1, 0),
+    ).animate(curvedAnimation)
+      ..addListener(() {
+        setState(() {});
+      })
+      ..addStatusListener((status) {
+        if (status == AnimationStatus.completed) {
+          controller.reverse();
+        }
+      });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(title: Text('Curved Animation')),
+      body: Column(
+        children: [
+          SizedBox(height: 20.0),
+          Text(
+            'Select Curve for forward motion',
+            style: Theme.of(context).textTheme.title,
+          ),
+          DropdownButton<CurveChoice>(
+            items: curves.map((curve) {
+              return DropdownMenuItem<CurveChoice>(
+                  value: curve, child: Text(curve.name));
+            }).toList(),
+            onChanged: (newCurve) {
+              setState(() {
+                selectedForwardCurve = newCurve;
+                curvedAnimation.curve = selectedForwardCurve.curve;
+              });
+            },
+            value: selectedForwardCurve,
+          ),
+          SizedBox(height: 15.0),
+          Text(
+            'Select Curve for reverse motion',
+            style: Theme.of(context).textTheme.title,
+          ),
+          DropdownButton<CurveChoice>(
+            items: curves.map((curve) {
+              return DropdownMenuItem<CurveChoice>(
+                  value: curve, child: Text(curve.name));
+            }).toList(),
+            onChanged: (newCurve) {
+              setState(() {
+                selectedReverseCurve = newCurve;
+                curvedAnimation.reverseCurve = selectedReverseCurve.curve;
+              });
+            },
+            value: selectedReverseCurve,
+          ),
+          SizedBox(height: 35.0),
+          Transform.rotate(
+            angle: animationRotation.value,
+            child: Center(
+              child: Container(
+                child: FlutterLogo(
+                  size: 100,
+                ),
+              ),
+            ),
+          ),
+          SizedBox(height: 35.0),
+          FractionalTranslation(
+            translation: animationTranslation.value,
+            child: Container(
+              child: FlutterLogo(
+                size: 100,
+              ),
+            ),
+          ),
+          SizedBox(height: 25.0),
+          RaisedButton(
+              onPressed: () {
+                controller.forward();
+              },
+              child: Text('Animate')),
```

Comment:
Add a comma in between the two parentheses on the end of this line as well.

Reply:
Although I am inserting a comma between the parenthesis ,I would like to quote @domesticmouse's reply from an earlier conversation  
> I'm not sure that change was required, this child `Text` element is quite small, and having it inline reduces the amount of vertical space consumed by the build method.